modepy/nodes.py

```diff
@@ -366,26 +366,30 @@ def tensor_product_nodes(
 
         *dims_or_nodes* can contain nodes of general size ``(dims, nnodes)``,
         not only one dimensional nodes.
+
+    .. versionchanged:: 2022.1
+
+        The node ordering changed once again, and it is now accessible, via
+        :func:`modepy.tools.reshape_array_for_tensor_product_space`.
     """
     if isinstance(dims_or_nodes, int):
         assert nodes_1d is not None
-        nodes: Sequence[np.ndarray] = [nodes_1d.reshape(1, -1)] * dims_or_nodes
+        nodesets: Sequence[np.ndarray] = [nodes_1d.reshape(1, -1)] * dims_or_nodes
```

Comment:
The first version of this PR made that assert into an exception with a nice message. Worth adding it back?

Reply:
https://github.com/inducer/modepy/compare/ef309cf2dd6e479823af74348b42144a75d6037a..993ee847d6096bf265d998ab89229bb6927e3a4b